src/resources/License/License.ts

```diff
@@ -4,8 +4,8 @@ import {LicenseModel} from './LicenseInterfaces';
 import API from '../../APICore';
 
 export default class License extends Resource {
-    static getBaseUrl = (sectionName: LicenseSection) =>
-        `/rest/organizations/${API.orgPlaceholder}/license/${sectionName}`;
+    static getBaseUrl = (sectionName?: LicenseSection) =>
+        `/rest/organizations/${API.orgPlaceholder}/license${sectionName ? `/${sectionName}` : ''}`;
```

Comment:
I'm not sure having nested backticks is legit. To simplify things we could simply remove the section name part from the baseUrl.
```suggestion
    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/license`;
```

And then in the individual methods simply add the sectionName where needed
```ts
return this.api.get<LicenseModel>(`${License.baseUrl}/${sectionName}`);
```

Reply:
I did it this way to avoid a trailing `/` but if its unnecessary I'll redo it